Build Failure With Maven 3.8.5

I'm getting a build failure after updating to a newer version of maven that came with fedora 37.

```
[ERROR] Failed to execute goal io.sundr:sundr-maven-plugin:0.40.1:generate-bom (default) on project parent: Error building generated bom:feign-bom: Could not create ReactorContext. org.apache.maven.session.scope.internal.SessionScope.memento() -> [Help 1]
```

``` 
mvn -version     
Apache Maven 3.8.5 (Red Hat 3.8.5-3)
Maven home: /usr/share/maven
Java version: 11.0.17-ea, vendor: Red Hat, Inc., runtime: /usr/lib/jvm/java-11-openjdk-11.0.17.0.1-0.1.ea.fc37.x86_64
Default locale: en_US, platform encoding: UTF-8
OS name: "linux", version: "6.0.7-301.fc37.x86_64", arch: "amd64", family: "unix"
```

Does not fail if using the mvn wrapper:
``` 
./mvnw -version
Apache Maven 3.5.4 (1edded0938998edf8bf061f1ceb3cfdeccf443fe; 2018-06-17T12:33:14-06:00)
Maven home: /home/jkomoroski/.m2/wrapper/dists/apache-maven-3.5.4-bin/jl1qqhdeineh9qg83dtj7i91c/apache-maven-3.5.4
Java version: 11.0.17-ea, vendor: Red Hat, Inc., runtime: /usr/lib/jvm/java-11-openjdk-11.0.17.0.1-0.1.ea.fc37.x86_64
Default locale: en_US, platform encoding: UTF-8
OS name: "linux", version: "6.0.7-301.fc37.x86_64", arch: "amd64", family: "unix"
```
It seems to me that ` io.sundr:sundr-maven-plugin:0.40.1` is very out of date.

--- a/pom.xml
+++ b/pom.xml
@@ -104,7 +104,7 @@
     <maven-bundle-plugin.version>5.1.8</maven-bundle-plugin.version>
     <centralsync-maven-plugin.version>0.1.1</centralsync-maven-plugin.version>
     <maven-surefire-plugin.version>2.22.2</maven-surefire-plugin.version>
-    <bom-generator.version>0.40.1</bom-generator.version>
+    <bom-generator.version>0.93.1</bom-generator.version>
     <bom.template.file.path>file://${project.basedir}/src/config/bom.xml</bom.template.file.path>
     <maven-scm-plugin.version>1.13.0</maven-scm-plugin.version>
     <maven-versions-plugin.version>2.13.0</maven-versions-plugin.version>
@@ -717,19 +717,6 @@
             </goals>
           </execution>
         </executions>
-        <dependencies>
-          <dependency>
-            <groupId>io.sundr</groupId>
-            <artifactId>sundr-codegen</artifactId>
-            <version>${bom-generator.version}</version>
-            <exclusions>
-              <exclusion>
-                <groupId>com.sun</groupId>
-                <artifactId>tools</artifactId>
-              </exclusion>
-            </exclusions>
-          </dependency>
-        </dependencies>
       </plugin>
       <plugin>
         <groupId>org.codehaus.mojo</groupId>

